packages/shared/utils/base64.ts

```diff
@@ -17,14 +17,13 @@
 export type Base64urlString = string;
 
 // Sourced from https://github.com/github/webauthn-json
-export function base64urlToBuffer(
-  baseurl64String: Base64urlString
-): ArrayBuffer {
-  // Base64url to Base64
+// Converts Base64url to Base64
+function base64urlToBase64(baseurl64String: Base64urlString): string {
```

Comment:
Is it worth writing our own test file? It seems like a lot of additional work to reinvent the wheel. Why not just use this optimized and tested library?

https://github.com/niklasvh/base64-arraybuffer

Reply:
makes sense, but we already have our own various implementation of `encode(buffer)` and i think it's because it was faster the way we do it, and we were developing it around the same time so now we have like two versions. We should've also tested it but we didn't (our bad :sweat_smile: ). 

We are bringing in this library just b/c we need the `decode` as far as i can tell so it feels weird to use a library now. We can get inspiration from their test files and incorporate it to our own (that's what i would've done). Or if we really want to use this library, then we use this and clean up our implementation of it. But in my experience here, something as small as this, we just copy the source code and credit the person.

Reply:
I went ahead and confirmed which are the fastest by running some tests on jsbench.me:
- The `base64-arraybuffer` library's implementation of base64 --> arraybuffer is ~50% faster: https://jsbench.me/vkkz8ly9os/1
- The fromCharCode (our current implementation) of arraybuffer --> Base64 is ~50% faster: https://jsbench.me/krkz8mf04o/1

I'll just add the source and test files from `base64-arraybuffer`, swap out our better implementation for `encode`, and use their test suite.